include/swift/SIL/SILConstants.h

```diff
@@ -57,6 +57,19 @@ enum class UnknownReason {
   Trap,
 };
 
+/// Determines how to substitute the function's generic arguments when
+/// evaluating an apply of the function.
+enum class FunctionSubConvention {
```

Comment:
Please name it something like `SubstitutionConvention`. "SubConvention" is very unclear to begin with -- subconvention sounds like a convention inside another convention.

Reply:
done